taf/updater/updater.py

```diff
@@ -739,7 +739,7 @@ def _commits_ret(commits, existing_repo, update_successful):
         )
     except Exception as e:
 
-        if not existing_repo:
+        if not existing_repo and not only_validate:
```

Comment:
Feels like something's wrong here if it's possible for `existing_repo` can be `None` and `only_validate` `True`. We only validate existing repos. What is deleting existing repos?

Reply:
Correct, removing that